test/cpp_extensions/open_registration_extension/torch_openreg/third_party/openreg/tests/event_tests.cpp

```diff
@@ -70,6 +94,60 @@ TEST_F(EventTest, EventElapsedTime) {
   EXPECT_EQ(orEventDestroy(end), orSuccess);
 }
 
+// TODO: recording events to a stream is not allowed
+// if the stream and the event are not on the same device
+// Uncomment this test case after the issue is fixed.
+// see #167819
+// TEST_F(EventTest, EventElapsedTimeDifferentDevicesFails) {
```

Comment:
Thank you for updating.

`GTest` have provided builtin capabilities to disable testcases temporarily, we can add `DISABLED_` prefix to `EventElapsedTimeDifferentDevicesFails`


```suggestion
// TEST_F(EventTest, DISABLED_EventElapsedTimeDifferentDevicesFails) {
```

Reply:
Thanks